pkg/scheduler/cache/node_info.go

```diff
@@ -75,7 +83,7 @@ type NodeInfo struct {
 
 	// Whenever NodeInfo changes, generation is bumped.
 	// This is used to avoid cloning it if the object didn't change.
-	generation int64
+	Generation int64
```

Comment:
Generation is public and easy to change.
(In the future, other fields should also be public.The current revision is just to avoid changing the code of other parts as much as possible. Just as misterikkit said,https://github.com/kubernetes/kubernetes/pull/68968#discussion_r220752522)

Reply:
Yeah, We should add a task in phase 2 to come back and clean this up.

Reply:
@bsalamat pointed out that allowing others to write to this field could cause errors. Maybe it should be hidden behind a getter?